salesforce.sgnl.yaml

```diff
@@ -8,18 +8,15 @@ defaultSyncFrequency: HOURLY
 defaultSyncMinInterval: 1
 defaultApiCallFrequency: SECONDLY
 defaultApiCallMinInterval: 1
-adapter:
-  hostname: "adapter.sgnl.svc.cluster.local"
-  port: 80
-  config: "ewoJInZlcnNpb24iOiAxLAoJImNvbmZpZyI6IHsKCQkicmVxdWVzdFRpbWVvdXQiOiAiMTBzIiwKCQkic2FsZXNmb3JjZSI6IHsiYXBpVmVyc2lvbiI6IjU2LjAifQoJfQp9"
-  apiVersion: "V1"
-  disableTls: true
-  auth:
-    - oAuth2ClientCredentials:
-        clientId: "{{Input Required: Client-ID}}"
-        tokenUrl: "https://{{Input Required: Tenant-Name}}.my.salesforce.com/services/oauth2/token?grant_type=client_credentials"
-        authStyle: AutoDetect
-        clientSecret: "{{Input Required: Client-Secret}}"
+type: "Salesforce-1.0.0"
+adapterConfig: "ewoJInZlcnNpb24iOiAxLAoJImNvbmZpZyI6IHsKCQkicmVxdWVzdFRpbWVvdXQiOiAiMTBzIiwKCQkic2FsZXNmb3JjZSI6IHsiYXBpVmVyc2lvbiI6IjU2LjAifQoJfQp9"
```

Comment:
Please add the following as a single-line comment here as an example of what it looks like:
```
{
	"version": 1,
	"config": {
		"requestTimeout": "10s",
		"salesforce": {"apiVersion":"56.0"}
	}
}
```